### ISSUE TITLE
Support export of project to PHP web host

### ISSUE THREAD

**Issue by davidfstr** [davidfstr/Crystal-Web-Archiver#178]
It would be nice to be able to share a downloaded website with the rest of the world at an online location that can be visited by a web browser.

The second cheapest type of web hosting, after [static web hosting](https://github.com/davidfstr/Crystal-Web-Archiver/issues/41), is PHP web hosting.

A PHP web host would definitely allow a downloaded site to be served with full fidelity. In particular all HTTP headers could be served correctly.

**Comment by davidfstr** [comment]
I'm probably going to always prefer [using a static file server](https://github.com/davidfstr/Crystal-Web-Archiver/issues/41) over any kind of dynamic web server (including PHP) to host since static file servers have pretty much zero operational overhead.

If I really wanted to fix the HTTP headers being inaccurate, I suspect I could use a Service Worker to fixup the headers after a site was loaded initially. But that's an entirely separate feature to develop...

Long story short, I don't think I care enough about this feature to implement it myself. Closing.
